requirements.txt

```diff
@@ -1 +1 @@
-monai>=1.0.1
+monai>=1.2.0rc5
```

Comment:
I think not all the bundles in model-zoo use the latest MONAI? Why upgrade this requirement for tests?

Thanks.

Reply:
thanks, I reverted this change, and add another monai requirement in `requirements-dev.txt`